index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Project Demo</title>
+    <script src="./js/jquery-3.3.1.min.js"></script>
```

Comment:
This file has not been pushed up to the repository, therefore jQuery is not linked and none of your JavaScript works.